maxim/tests/test_decorators.py

```
@@ -13,36 +13,41 @@
+        # This is a hack to ensure that the Maxim instance is not cached
+        if hasattr(Maxim, "_instance"):
+            delattr(Maxim, "_instance")
+        self.logger = Maxim().logger()
+        self.mock_writer = inject_mock_writer(self.logger)
 
     def test_openai_chat_one(self):
         print("here")
 
-        @span("secondTest")
+        @span(name="secondTest")
         def secondTest():
```

Ruff (0.11.9)

Missing return type annotation for private function `secondTest`

Add return type annotation: `None`

(ANN202)